specs.html

```diff
@@ -63,7 +63,10 @@ <h1>Using GitHub for Spec Work</h1>
         We can then add it to the repository.
       </p>
       <pre>git add index.html
-git commit -m "Add empty doc."</pre>
+git commit -m "chore: Add empty doc."</pre>
+      <p>
+        Tip: consider using the conventions <q><a href="http://karma-runner.github.io/1.0/dev/git-commit-msg.html">Git Commit Msg</a></q>.
+      </p>
```

Comment:
SGTM, but maybe link to repos that have already adopted these conventions for W3C spec work.

(also, if we want to promote it, documenting them is necessary but clearly not sufficient :)